pyalgotrade/optimizer/xmlrpcserver.py

```diff
@@ -89,6 +87,9 @@ def __init__(self, paramSource, resultSinc, barFeed, address, port, autoStop=Tru
         self.register_function(self.getNextJob, 'getNextJob')
         self.register_function(self.pushJobResults, 'pushJobResults')
 
+    def getBatchSize(self):
```

Comment:
I actually just want to let users being able to choose their batchSize, do not have opinions about getBatchSize() method. But I do not quite understand the code coverage problem, could you please explain a bit more? Sorry for inconvenience!